### PR TITLE
[IMP] stock_request_picking_type: default picking type on SROs

### PR DESCRIPTION
**Pull request by RLeeGML**

### CODE REVIEW

File: stock_request_picking_type/models/stock_request_order.py [hunk 1]
```diff
@@ -24,6 +25,7 @@ def _get_default_picking_type(self):
                             False,
                         ],
                     ),
+                    ("warehouse_id", "=", False),
```
Comment (bodedra) [inline stock_request_picking_type/models/stock_request_order.py]: Would you please provide explanation for this change?

Comment (RLeeGML) [reply stock_request_picking_type/models/stock_request_order.py]: If multi-warehouse configuration setting is not enabled, the stock.picking.type is created with no warehouse_id. If the user does not have multi-warehouse enabled, this change will allow the default picking_type_id to be assigned properly

**Comment by bodedra** [comment]
@ps-tubtim Does this fix work for you? If yes, would you please merge it.

**Comment by ps-tubtim** [comment]
@RLeeOSI Thank you. I merge this PR on https://github.com/OCA/stock-logistics-warehouse/pull/1010.

**Comment by ps-tubtim** [comment]
@RLeeOSI Thank you for your help. I already merged this PR on OCA#1010.
Close this PR.

**Pull request closed without merging**